prow/plugins/trigger/generic-comment.go

```diff
@@ -183,3 +231,47 @@ func getContexts(combinedStatus *github.CombinedStatus) (sets.String, sets.Strin
 	}
 	return failedContexts, allContexts
 }
+
+func markdownList(names []string) string {
+	var listBuilder strings.Builder
+	for _, name := range names {
+		listBuilder.WriteString(fmt.Sprintf("\n* `%s`", name))
+	}
+	return listBuilder.String()
+}
+
+func buildHelpComment(githubClient githubClient, body, org, repo, branch string, number int, presubmits []config.Presubmit, HTMLURL, user, note string, logger *logrus.Entry) error {
+	available, err := availablePresubmits(githubClient, body, org, repo, branch, number, presubmits, logger)
+	if err != nil {
+		return err
+	}
+	var resp string
+	if len(available) > 0 {
+		resp = fmt.Sprintf("%sThe following jobs can be triggered by using their name after a `/test` message: %s", note, markdownList(available))
+	} else {
+		resp = fmt.Sprintf("%sNo presubmit jobs available for %s/%s=%s", note, org, repo, branch)
```

Comment:
nit: don't we idiomatically write `org/repo@branch` instead of `org/repo=branch`?

Reply:
Thank you for the review. Fixed.